info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-25T12:50:21.870Z"}
error: Could not connect to the database. Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/ {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"ReplicaSetNoPrimary"},"stack":"MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/\n    at NativeConnection.Connection.openUri (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:351:10\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (D:\\project1_googleapi\\src\\config\\/database.js:12:20)\n    at tryCatch (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:24)\n    at _next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:9)\n    at D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:27:7\n    at new Promise (<anonymous>)\n    at apply (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:19:12)\n    at database (D:\\project1_googleapi\\src\\config\\/database.js:4:15)\n    at Object.<anonymous> (D:\\project1_googleapi\\src\\/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Module._compile (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:117:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Object.newLoader [as .js] (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:121:7)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.runMain (node:internal/modules/run_main:135:12)\n    at Object.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\node\\src\\_babel-node.ts:222:12)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)\n    at node:internal/main/run_main_module:28:49","timestamp":"2024-01-25T12:50:52.006Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-25T12:52:57.773Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-25T12:53:10.704Z"}
error: Could not connect to the database. Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/ {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"ReplicaSetNoPrimary"},"stack":"MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/\n    at NativeConnection.Connection.openUri (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:351:10\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (D:\\project1_googleapi\\src\\config\\/database.js:12:20)\n    at tryCatch (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:24)\n    at _next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:9)\n    at D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:27:7\n    at new Promise (<anonymous>)\n    at apply (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:19:12)\n    at database (D:\\project1_googleapi\\src\\config\\/database.js:4:15)\n    at Object.<anonymous> (D:\\project1_googleapi\\src\\/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Module._compile (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:117:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Object.newLoader [as .js] (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:121:7)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.runMain (node:internal/modules/run_main:135:12)\n    at Object.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\node\\src\\_babel-node.ts:222:12)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)\n    at node:internal/main/run_main_module:28:49","timestamp":"2024-01-25T12:53:40.804Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-25T12:59:34.889Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-25T12:59:39.014Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-25T12:59:50.301Z"}
error: 
    status - 500 
    message - ValidationError: "first_name" is required 
    url - /api/v1/users 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-25T13:00:08.413Z"}
error: 
    status - 500 
    message - ValidationError: "emailid" is required 
    url - /api/v1/users 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-25T13:00:15.854Z"}
error: Could not connect to the database. Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/ {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"ReplicaSetNoPrimary"},"stack":"MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/\n    at NativeConnection.Connection.openUri (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:351:10\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (D:\\project1_googleapi\\src\\config\\/database.js:12:20)\n    at tryCatch (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:24)\n    at _next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:9)\n    at D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:27:7\n    at new Promise (<anonymous>)\n    at apply (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:19:12)\n    at database (D:\\project1_googleapi\\src\\config\\/database.js:4:15)\n    at Object.<anonymous> (D:\\project1_googleapi\\src\\/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Module._compile (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:117:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Object.newLoader [as .js] (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:121:7)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.runMain (node:internal/modules/run_main:135:12)\n    at Object.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\node\\src\\_babel-node.ts:222:12)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)\n    at node:internal/main/run_main_module:28:49","timestamp":"2024-01-25T13:00:20.403Z"}
error: 
    status - 500 
    message - MongooseError: Operation `users.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (D:\project1_googleapi\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:198:23)
    at listOnTimeout (node:internal/timers:573:17)
    at processTimers (node:internal/timers:514:7) 
    url - /api/v1/users 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-25T13:01:12.205Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-25T13:04:09.925Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-25T13:04:19.624Z"}
error: 
    status - 500 
    message - MongooseError: Operation `users.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (D:\project1_googleapi\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:198:23)
    at listOnTimeout (node:internal/timers:573:17)
    at processTimers (node:internal/timers:514:7) 
    url - /api/v1/users 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-25T13:04:32.736Z"}
error: Could not connect to the database. Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/ {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"ReplicaSetNoPrimary"},"stack":"MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/\n    at NativeConnection.Connection.openUri (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:351:10\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (D:\\project1_googleapi\\src\\config\\/database.js:12:20)\n    at tryCatch (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:24)\n    at _next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:9)\n    at D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:27:7\n    at new Promise (<anonymous>)\n    at apply (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:19:12)\n    at database (D:\\project1_googleapi\\src\\config\\/database.js:4:15)\n    at Object.<anonymous> (D:\\project1_googleapi\\src\\/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Module._compile (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:117:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Object.newLoader [as .js] (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:121:7)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.runMain (node:internal/modules/run_main:135:12)\n    at Object.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\node\\src\\_babel-node.ts:222:12)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)\n    at node:internal/main/run_main_module:28:49","timestamp":"2024-01-25T13:04:49.719Z"}
error: 
    status - 500 
    message - MongooseError: Operation `users.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (D:\project1_googleapi\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:198:23)
    at listOnTimeout (node:internal/timers:573:17)
    at processTimers (node:internal/timers:514:7) 
    url - /api/v1/users 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-25T13:07:37.935Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-25T13:10:53.050Z"}
error: Could not connect to the database. Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/ {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"ReplicaSetNoPrimary"},"stack":"MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/\n    at NativeConnection.Connection.openUri (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:351:10\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (D:\\project1_googleapi\\src\\config\\/database.js:12:20)\n    at tryCatch (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:24)\n    at _next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:9)\n    at D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:27:7\n    at new Promise (<anonymous>)\n    at apply (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:19:12)\n    at database (D:\\project1_googleapi\\src\\config\\/database.js:4:15)\n    at Object.<anonymous> (D:\\project1_googleapi\\src\\/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Module._compile (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:117:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Object.newLoader [as .js] (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:121:7)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.runMain (node:internal/modules/run_main:135:12)\n    at Object.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\node\\src\\_babel-node.ts:222:12)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)\n    at node:internal/main/run_main_module:28:49","timestamp":"2024-01-25T13:11:23.175Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-25T13:11:55.868Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-25T13:11:58.196Z"}
error: Could not connect to the database. Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/ {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"ReplicaSetNoPrimary"},"stack":"MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/\n    at NativeConnection.Connection.openUri (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:351:10\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (D:\\project1_googleapi\\src\\config\\/database.js:12:20)\n    at tryCatch (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:24)\n    at _next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:9)\n    at D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:27:7\n    at new Promise (<anonymous>)\n    at apply (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:19:12)\n    at database (D:\\project1_googleapi\\src\\config\\/database.js:4:15)\n    at Object.<anonymous> (D:\\project1_googleapi\\src\\/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Module._compile (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:117:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Object.newLoader [as .js] (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:121:7)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.runMain (node:internal/modules/run_main:135:12)\n    at Object.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\node\\src\\_babel-node.ts:222:12)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)\n    at node:internal/main/run_main_module:28:49","timestamp":"2024-01-25T13:12:28.346Z"}
error: 
    status - 500 
    message - MongooseError: Operation `users.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (D:\project1_googleapi\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:198:23)
    at listOnTimeout (node:internal/timers:573:17)
    at processTimers (node:internal/timers:514:7) 
    url - /api/v1/users 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-25T13:12:36.899Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-25T13:13:18.922Z"}
error: 
    status - 500 
    message - MongooseError: Operation `users.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (D:\project1_googleapi\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:198:23)
    at listOnTimeout (node:internal/timers:573:17)
    at processTimers (node:internal/timers:514:7) 
    url - /api/v1/users 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-25T13:13:33.062Z"}
error: Could not connect to the database. Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/ {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"ReplicaSetNoPrimary"},"stack":"MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/\n    at NativeConnection.Connection.openUri (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:351:10\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (D:\\project1_googleapi\\src\\config\\/database.js:12:20)\n    at tryCatch (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:24)\n    at _next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:9)\n    at D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:27:7\n    at new Promise (<anonymous>)\n    at apply (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:19:12)\n    at database (D:\\project1_googleapi\\src\\config\\/database.js:4:15)\n    at Object.<anonymous> (D:\\project1_googleapi\\src\\/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Module._compile (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:117:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Object.newLoader [as .js] (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:121:7)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.runMain (node:internal/modules/run_main:135:12)\n    at Object.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\node\\src\\_babel-node.ts:222:12)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)\n    at node:internal/main/run_main_module:28:49","timestamp":"2024-01-25T13:13:49.088Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-25T13:14:05.825Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-25T13:14:08.339Z"}
error: Could not connect to the database. Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/ {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"ReplicaSetNoPrimary"},"stack":"MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/\n    at NativeConnection.Connection.openUri (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:351:10\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (D:\\project1_googleapi\\src\\config\\/database.js:12:20)\n    at tryCatch (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:24)\n    at _next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:9)\n    at D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:27:7\n    at new Promise (<anonymous>)\n    at apply (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:19:12)\n    at database (D:\\project1_googleapi\\src\\config\\/database.js:4:15)\n    at Object.<anonymous> (D:\\project1_googleapi\\src\\/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Module._compile (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:117:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Object.newLoader [as .js] (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:121:7)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.runMain (node:internal/modules/run_main:135:12)\n    at Object.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\node\\src\\_babel-node.ts:222:12)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)\n    at node:internal/main/run_main_module:28:49","timestamp":"2024-01-25T13:14:38.545Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-25T13:15:23.409Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-25T13:15:33.613Z"}
error: 
    status - 500 
    message - MongooseError: Operation `users.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (D:\project1_googleapi\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:198:23)
    at listOnTimeout (node:internal/timers:573:17)
    at processTimers (node:internal/timers:514:7) 
    url - /api/v1/users 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-25T13:15:50.873Z"}
error: Could not connect to the database. Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/ {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"ReplicaSetNoPrimary"},"stack":"MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/\n    at NativeConnection.Connection.openUri (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:351:10\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (D:\\project1_googleapi\\src\\config\\/database.js:12:20)\n    at tryCatch (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:24)\n    at _next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:9)\n    at D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:27:7\n    at new Promise (<anonymous>)\n    at apply (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:19:12)\n    at database (D:\\project1_googleapi\\src\\config\\/database.js:4:15)\n    at Object.<anonymous> (D:\\project1_googleapi\\src\\/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Module._compile (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:117:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Object.newLoader [as .js] (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:121:7)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.runMain (node:internal/modules/run_main:135:12)\n    at Object.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\node\\src\\_babel-node.ts:222:12)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)\n    at node:internal/main/run_main_module:28:49","timestamp":"2024-01-25T13:16:09.777Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-25T13:16:47.952Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-25T13:17:02.493Z"}
error: 
    status - 500 
    message - MongooseError: Operation `users.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (D:\project1_googleapi\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:198:23)
    at listOnTimeout (node:internal/timers:573:17)
    at processTimers (node:internal/timers:514:7) 
    url - /api/v1/users 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-25T13:17:20.488Z"}
error: Could not connect to the database. Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/ {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"ReplicaSetNoPrimary"},"stack":"MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/\n    at NativeConnection.Connection.openUri (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:351:10\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (D:\\project1_googleapi\\src\\config\\/database.js:12:20)\n    at tryCatch (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:24)\n    at _next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:9)\n    at D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:27:7\n    at new Promise (<anonymous>)\n    at apply (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:19:12)\n    at database (D:\\project1_googleapi\\src\\config\\/database.js:4:15)\n    at Object.<anonymous> (D:\\project1_googleapi\\src\\/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Module._compile (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:117:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Object.newLoader [as .js] (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:121:7)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.runMain (node:internal/modules/run_main:135:12)\n    at Object.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\node\\src\\_babel-node.ts:222:12)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)\n    at node:internal/main/run_main_module:28:49","timestamp":"2024-01-25T13:17:32.622Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-25T13:17:45.301Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-25T13:17:47.916Z"}
error: 
    status - 500 
    message - MongooseError: Operation `users.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (D:\project1_googleapi\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:198:23)
    at listOnTimeout (node:internal/timers:573:17)
    at processTimers (node:internal/timers:514:7) 
    url - /api/v1/users 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-25T13:18:02.238Z"}
error: Could not connect to the database. Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/ {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"ReplicaSetNoPrimary"},"stack":"MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/\n    at NativeConnection.Connection.openUri (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:351:10\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (D:\\project1_googleapi\\src\\config\\/database.js:12:20)\n    at tryCatch (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:24)\n    at _next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:9)\n    at D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:27:7\n    at new Promise (<anonymous>)\n    at apply (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:19:12)\n    at database (D:\\project1_googleapi\\src\\config\\/database.js:4:15)\n    at Object.<anonymous> (D:\\project1_googleapi\\src\\/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Module._compile (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:117:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Object.newLoader [as .js] (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:121:7)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.runMain (node:internal/modules/run_main:135:12)\n    at Object.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\node\\src\\_babel-node.ts:222:12)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)\n    at node:internal/main/run_main_module:28:49","timestamp":"2024-01-25T13:18:18.016Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-25T13:18:52.706Z"}
error: 
    status - 500 
    message - MongooseError: Operation `users.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (D:\project1_googleapi\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:198:23)
    at listOnTimeout (node:internal/timers:573:17)
    at processTimers (node:internal/timers:514:7) 
    url - /api/v1/users 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-25T13:19:15.227Z"}
error: Could not connect to the database. Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/ {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"ReplicaSetNoPrimary"},"stack":"MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/\n    at NativeConnection.Connection.openUri (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:351:10\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (D:\\project1_googleapi\\src\\config\\/database.js:12:20)\n    at tryCatch (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:24)\n    at _next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:9)\n    at D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:27:7\n    at new Promise (<anonymous>)\n    at apply (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:19:12)\n    at database (D:\\project1_googleapi\\src\\config\\/database.js:4:15)\n    at Object.<anonymous> (D:\\project1_googleapi\\src\\/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Module._compile (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:117:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Object.newLoader [as .js] (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:121:7)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.runMain (node:internal/modules/run_main:135:12)\n    at Object.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\node\\src\\_babel-node.ts:222:12)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)\n    at node:internal/main/run_main_module:28:49","timestamp":"2024-01-25T13:19:24.845Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-25T13:19:32.753Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-25T13:19:57.511Z"}
error: 
    status - 500 
    message - MongooseError: Operation `users.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (D:\project1_googleapi\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:198:23)
    at listOnTimeout (node:internal/timers:573:17)
    at processTimers (node:internal/timers:514:7) 
    url - /api/v1/newuser1 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-25T13:20:24.536Z"}
error: Could not connect to the database. Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/ {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"ReplicaSetNoPrimary"},"stack":"MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/\n    at NativeConnection.Connection.openUri (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:351:10\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (D:\\project1_googleapi\\src\\config\\/database.js:12:20)\n    at tryCatch (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:24)\n    at _next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:9)\n    at D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:27:7\n    at new Promise (<anonymous>)\n    at apply (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:19:12)\n    at database (D:\\project1_googleapi\\src\\config\\/database.js:4:15)\n    at Object.<anonymous> (D:\\project1_googleapi\\src\\/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Module._compile (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:117:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Object.newLoader [as .js] (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:121:7)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.runMain (node:internal/modules/run_main:135:12)\n    at Object.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\node\\src\\_babel-node.ts:222:12)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)\n    at node:internal/main/run_main_module:28:49","timestamp":"2024-01-25T13:20:27.661Z"}
error: 
    status - 500 
    message - MongooseError: Operation `users.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (D:\project1_googleapi\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:198:23)
    at listOnTimeout (node:internal/timers:573:17)
    at processTimers (node:internal/timers:514:7) 
    url - /api/v1/newuser1 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-25T13:21:16.649Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-25T13:25:11.929Z"}
info: Connected to the database. {"timestamp":"2024-01-25T13:25:11.950Z"}
error: 
    status - 500 
    message - ValidationError: "first_name" is required 
    url - /api/v1/newuser1 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-25T13:26:31.862Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-27T08:34:56.203Z"}
info: Connected to the database. {"timestamp":"2024-01-27T08:34:56.233Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-27T09:25:29.462Z"}
info: Connected to the database. {"timestamp":"2024-01-27T09:25:29.506Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-27T09:25:54.656Z"}
info: Connected to the database. {"timestamp":"2024-01-27T09:25:54.673Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-27T09:26:21.361Z"}
info: Connected to the database. {"timestamp":"2024-01-27T09:26:21.381Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-27T09:26:26.114Z"}
info: Connected to the database. {"timestamp":"2024-01-27T09:26:26.130Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-27T09:26:35.822Z"}
info: Connected to the database. {"timestamp":"2024-01-27T09:26:35.843Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-27T09:26:39.315Z"}
info: Connected to the database. {"timestamp":"2024-01-27T09:26:39.331Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-27T09:27:57.392Z"}
info: Connected to the database. {"timestamp":"2024-01-27T09:27:57.419Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-27T09:28:02.620Z"}
info: Connected to the database. {"timestamp":"2024-01-27T09:28:02.639Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-27T09:28:05.465Z"}
info: Connected to the database. {"timestamp":"2024-01-27T09:28:05.486Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-27T09:28:09.119Z"}
info: Connected to the database. {"timestamp":"2024-01-27T09:28:09.143Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-27T09:28:12.383Z"}
info: Connected to the database. {"timestamp":"2024-01-27T09:28:12.408Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-27T09:28:14.552Z"}
info: Connected to the database. {"timestamp":"2024-01-27T09:28:14.571Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-27T09:28:30.362Z"}
info: Connected to the database. {"timestamp":"2024-01-27T09:28:30.383Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-27T09:28:39.660Z"}
info: Connected to the database. {"timestamp":"2024-01-27T09:28:39.678Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-27T09:28:52.409Z"}
info: Connected to the database. {"timestamp":"2024-01-27T09:28:52.446Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-27T09:29:02.258Z"}
info: Connected to the database. {"timestamp":"2024-01-27T09:29:02.286Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-27T09:29:07.450Z"}
info: Connected to the database. {"timestamp":"2024-01-27T09:29:07.477Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-27T09:29:17.568Z"}
info: Connected to the database. {"timestamp":"2024-01-27T09:29:17.597Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-27T09:29:23.770Z"}
info: Connected to the database. {"timestamp":"2024-01-27T09:29:23.826Z"}
error: 
    status - 500 
    message - ValidationError: "first_name" is required 
    url - /api/v1/newuser1 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-27T09:29:44.391Z"}
error: 
    status - 500 
    message - ReferenceError: email is not defined
    at _callee5$ (D:\project1_googleapi\src\services\/user.service.js:37:47)
    at tryCatch (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at Object.apply (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12)
    at Object.getUser (D:\project1_googleapi\src\services\/user.service.js:36:21)
    at _callee2$ (D:\project1_googleapi\src\controllers\/user.controller.js:31:36)
    at tryCatch (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at apply (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12)
    at getUser (D:\project1_googleapi\src\controllers\/user.controller.js:29:21)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\project1_googleapi\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (D:\project1_googleapi\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:284:15
    at param (D:\project1_googleapi\node_modules\express\lib\router\index.js:365:14)
    at param (D:\project1_googleapi\node_modules\express\lib\router\index.js:376:14)
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:421:3)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\project1_googleapi\node_modules\express\lib\router\index.js:175:3)
    at router (D:\project1_googleapi\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\project1_googleapi\node_modules\express\lib\router\index.js:175:3)
    at router (D:\project1_googleapi\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at logger (D:\project1_googleapi\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at jsonParser (D:\project1_googleapi\node_modules\body-parser\lib\types\json.js:110:7)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at urlencodedParser (D:\project1_googleapi\node_modules\body-parser\lib\types\urlencoded.js:91:7) 
    url - /api/v1/newuser1/65b4cd0d1c94330f8c97cec6 
    method - GET 
    IP - ::1
   {"timestamp":"2024-01-27T09:30:01.496Z"}
error: 
    status - 500 
    message - ReferenceError: email is not defined
    at _callee5$ (D:\project1_googleapi\src\services\/user.service.js:37:47)
    at tryCatch (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at Object.apply (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12)
    at Object.getUser (D:\project1_googleapi\src\services\/user.service.js:36:21)
    at _callee2$ (D:\project1_googleapi\src\controllers\/user.controller.js:31:36)
    at tryCatch (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at apply (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12)
    at getUser (D:\project1_googleapi\src\controllers\/user.controller.js:29:21)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\project1_googleapi\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (D:\project1_googleapi\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:284:15
    at param (D:\project1_googleapi\node_modules\express\lib\router\index.js:365:14)
    at param (D:\project1_googleapi\node_modules\express\lib\router\index.js:376:14)
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:421:3)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\project1_googleapi\node_modules\express\lib\router\index.js:175:3)
    at router (D:\project1_googleapi\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\project1_googleapi\node_modules\express\lib\router\index.js:175:3)
    at router (D:\project1_googleapi\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at logger (D:\project1_googleapi\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at D:\project1_googleapi\node_modules\body-parser\lib\read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:206:9)
    at invokeCallback (D:\project1_googleapi\node_modules\raw-body\index.js:231:16)
    at done (D:\project1_googleapi\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (D:\project1_googleapi\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:518:28)
    at IncomingMessage.emit (node:domain:488:12) 
    url - /api/v1/newuser1/65b4cd0d1c94330f8c97cec6 
    method - GET 
    IP - ::1
   {"timestamp":"2024-01-27T09:31:49.815Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-27T09:32:18.428Z"}
info: Connected to the database. {"timestamp":"2024-01-27T09:32:18.450Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-27T09:32:53.715Z"}
info: Connected to the database. {"timestamp":"2024-01-27T09:32:53.747Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-27T09:32:56.411Z"}
info: Connected to the database. {"timestamp":"2024-01-27T09:32:56.428Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-27T09:46:43.402Z"}
info: Connected to the database. {"timestamp":"2024-01-27T09:46:43.423Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-27T09:47:56.840Z"}
info: Connected to the database. {"timestamp":"2024-01-27T09:47:56.860Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T18:03:19.403Z"}
info: Connected to the database. {"timestamp":"2024-01-28T18:03:19.432Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T18:03:31.600Z"}
info: Connected to the database. {"timestamp":"2024-01-28T18:03:31.623Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T18:04:52.228Z"}
info: Connected to the database. {"timestamp":"2024-01-28T18:04:52.253Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T18:06:08.007Z"}
info: Connected to the database. {"timestamp":"2024-01-28T18:06:08.031Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T18:06:12.041Z"}
info: Connected to the database. {"timestamp":"2024-01-28T18:06:12.060Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T18:07:22.502Z"}
info: Connected to the database. {"timestamp":"2024-01-28T18:07:22.521Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T18:10:39.693Z"}
info: Connected to the database. {"timestamp":"2024-01-28T18:10:39.710Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T18:11:32.926Z"}
info: Connected to the database. {"timestamp":"2024-01-28T18:11:32.955Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T18:11:35.332Z"}
info: Connected to the database. {"timestamp":"2024-01-28T18:11:35.354Z"}
error: 
    status - 500 
    message - ValidationError: "first_name" is required 
    url - /api/v1/user/registration 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-28T18:11:43.683Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T18:12:02.041Z"}
info: Connected to the database. {"timestamp":"2024-01-28T18:12:02.070Z"}
error: 
    status - 500 
    message - ValidationError: "lastName" is not allowed 
    url - /api/v1/user/registration 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-28T18:12:06.450Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T18:18:07.021Z"}
info: Connected to the database. {"timestamp":"2024-01-28T18:18:07.037Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T18:18:19.656Z"}
info: Connected to the database. {"timestamp":"2024-01-28T18:18:19.688Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T18:18:35.084Z"}
info: Connected to the database. {"timestamp":"2024-01-28T18:18:35.103Z"}
error: 
    status - 500 
    message - ValidationError: "email_id" is required 
    url - /api/v1/user/registration 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-28T18:18:38.450Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T18:18:52.441Z"}
info: Connected to the database. {"timestamp":"2024-01-28T18:18:52.472Z"}
error: 
    status - 500 
    message - TypeError: Cannot read properties of undefined (reading 'json')
    at _callee2$ (D:\project1_googleapi\src\services\/user.service.js:31:9)
    at tryCatch (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/user/registration 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-28T18:18:56.281Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T18:25:01.796Z"}
info: Connected to the database. {"timestamp":"2024-01-28T18:25:01.819Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T18:26:22.799Z"}
info: Connected to the database. {"timestamp":"2024-01-28T18:26:22.819Z"}
error: 
    status - 500 
    message - Error: User with the same email already exists
    at _callee2$ (D:\project1_googleapi\src\services\/user.service.js:18:11)
    at tryCatch (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/user/registration 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-28T18:26:28.555Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T18:27:26.731Z"}
info: Connected to the database. {"timestamp":"2024-01-28T18:27:26.749Z"}
error: 
    status - 500 
    message - Error: User with the same email already exists
    at _callee2$ (D:\project1_googleapi\src\services\/user.service.js:16:11)
    at tryCatch (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/user/registration 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-28T18:27:30.361Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T18:36:18.320Z"}
info: Connected to the database. {"timestamp":"2024-01-28T18:36:18.334Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T18:38:13.291Z"}
info: Connected to the database. {"timestamp":"2024-01-28T18:38:13.324Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T18:38:16.455Z"}
info: Connected to the database. {"timestamp":"2024-01-28T18:38:16.470Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T18:38:20.567Z"}
info: Connected to the database. {"timestamp":"2024-01-28T18:38:20.590Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T18:38:24.503Z"}
info: Connected to the database. {"timestamp":"2024-01-28T18:38:24.540Z"}
error: 
    status - 500 
    message - Error: User with the same email already exists
    at _callee2$ (D:\project1_googleapi\src\services\/user.service.js:16:11)
    at tryCatch (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/user/registration 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-28T18:38:40.044Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T18:39:33.007Z"}
info: Connected to the database. {"timestamp":"2024-01-28T18:39:33.023Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T18:39:35.176Z"}
info: Connected to the database. {"timestamp":"2024-01-28T18:39:35.190Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T18:39:39.769Z"}
info: Connected to the database. {"timestamp":"2024-01-28T18:39:39.786Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T18:39:50.215Z"}
info: Connected to the database. {"timestamp":"2024-01-28T18:39:50.228Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T18:39:53.812Z"}
info: Connected to the database. {"timestamp":"2024-01-28T18:39:53.825Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T18:44:36.288Z"}
info: Connected to the database. {"timestamp":"2024-01-28T18:44:36.303Z"}
error: 
    status - 500 
    message - Error: User with the same email already exists
    at _callee2$ (D:\project1_googleapi\src\services\/user.service.js:16:11)
    at tryCatch (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/user/registration 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-28T18:45:23.751Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T18:47:41.738Z"}
info: Connected to the database. {"timestamp":"2024-01-28T18:47:41.751Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T18:51:09.106Z"}
info: Connected to the database. {"timestamp":"2024-01-28T18:51:09.126Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T18:52:42.590Z"}
info: Connected to the database. {"timestamp":"2024-01-28T18:52:42.606Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T18:54:23.016Z"}
info: Connected to the database. {"timestamp":"2024-01-28T18:54:23.033Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T18:57:42.150Z"}
info: Connected to the database. {"timestamp":"2024-01-28T18:57:42.163Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T18:59:21.233Z"}
info: Connected to the database. {"timestamp":"2024-01-28T18:59:21.246Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T19:00:00.066Z"}
info: Connected to the database. {"timestamp":"2024-01-28T19:00:00.080Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T19:02:23.541Z"}
info: Connected to the database. {"timestamp":"2024-01-28T19:02:23.557Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T19:02:35.878Z"}
info: Connected to the database. {"timestamp":"2024-01-28T19:02:35.894Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T19:05:01.555Z"}
info: Connected to the database. {"timestamp":"2024-01-28T19:05:01.573Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T19:05:15.770Z"}
info: Connected to the database. {"timestamp":"2024-01-28T19:05:15.794Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T19:07:51.442Z"}
info: Connected to the database. {"timestamp":"2024-01-28T19:07:51.457Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T19:08:34.768Z"}
info: Connected to the database. {"timestamp":"2024-01-28T19:08:34.782Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T19:09:29.421Z"}
info: Connected to the database. {"timestamp":"2024-01-28T19:09:29.434Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T19:10:18.180Z"}
info: Connected to the database. {"timestamp":"2024-01-28T19:10:18.196Z"}
error: 
    status - 500 
    message - Error: User with the same email already exists
    at _callee2$ (D:\project1_googleapi\src\services\/user.service.js:16:11)
    at tryCatch (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/user/registration 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-28T19:11:13.109Z"}
error: 
    status - 500 
    message - Error: User with the same email already exists
    at _callee2$ (D:\project1_googleapi\src\services\/user.service.js:16:11)
    at tryCatch (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/user/registration 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-28T19:11:20.731Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T19:13:31.237Z"}
info: Connected to the database. {"timestamp":"2024-01-28T19:13:31.254Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T19:15:39.812Z"}
info: Connected to the database. {"timestamp":"2024-01-28T19:15:39.831Z"}
error: 
    status - 500 
    message - Error: User with the same email already exists
    at _callee2$ (D:\project1_googleapi\src\services\/user.service.js:17:11)
    at tryCatch (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/user/registration 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-28T19:15:43.666Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T19:17:14.783Z"}
info: Connected to the database. {"timestamp":"2024-01-28T19:17:14.797Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T19:21:47.042Z"}
info: Connected to the database. {"timestamp":"2024-01-28T19:21:47.057Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-28T19:22:00.095Z"}
info: Connected to the database. {"timestamp":"2024-01-28T19:22:00.111Z"}
error: 
    status - 500 
    message - Error: User with the same email already exists
    at _callee2$ (D:\project1_googleapi\src\services\/user.service.js:17:11)
    at tryCatch (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/user/registration 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-28T19:22:06.002Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-29T05:50:25.760Z"}
info: Connected to the database. {"timestamp":"2024-01-29T05:50:25.780Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-29T06:18:31.490Z"}
info: Connected to the database. {"timestamp":"2024-01-29T06:18:31.507Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-31T07:11:29.670Z"}
info: Connected to the database. {"timestamp":"2024-01-31T07:11:29.688Z"}
error: 
    status - 500 
    message - ValidationError: "firstName" is required 
    url - /api/v1/user/registration 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-31T07:12:59.359Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-31T07:14:30.099Z"}
info: Connected to the database. {"timestamp":"2024-01-31T07:14:30.130Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-31T07:14:54.255Z"}
info: Connected to the database. {"timestamp":"2024-01-31T07:14:54.276Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-31T07:15:01.748Z"}
info: Connected to the database. {"timestamp":"2024-01-31T07:15:01.767Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2024-01-31T07:15:39.761Z"}
info: Connected to the database. {"timestamp":"2024-01-31T07:15:39.780Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at D:\project1_googleapi\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (D:\project1_googleapi\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.verify (D:\project1_googleapi\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (D:\project1_googleapi\src\middlewares\/auth.middleware.js:22:32)
    at tryCatch (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at apply (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12)
    at userAuth (D:\project1_googleapi\src\middlewares\/auth.middleware.js:12:22)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\project1_googleapi\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (D:\project1_googleapi\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\project1_googleapi\node_modules\express\lib\router\index.js:175:3)
    at router (D:\project1_googleapi\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\project1_googleapi\node_modules\express\lib\router\index.js:175:3)
    at router (D:\project1_googleapi\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at logger (D:\project1_googleapi\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at D:\project1_googleapi\node_modules\body-parser\lib\read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:206:9)
    at invokeCallback (D:\project1_googleapi\node_modules\raw-body\index.js:231:16)
    at done (D:\project1_googleapi\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (D:\project1_googleapi\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:518:28)
    at IncomingMessage.emit (node:domain:488:12)
    at endReadableNT (node:internal/streams/readable:1696:12)
    at processTicksAndRejections (node:internal/process/task_queues:82:21) 
    url - /api/v1/user/profile 
    method - GET 
    IP - ::1
   {"timestamp":"2024-01-31T07:15:53.620Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at D:\project1_googleapi\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (D:\project1_googleapi\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.verify (D:\project1_googleapi\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (D:\project1_googleapi\src\middlewares\/auth.middleware.js:22:32)
    at tryCatch (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at apply (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12)
    at userAuth (D:\project1_googleapi\src\middlewares\/auth.middleware.js:12:22)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\project1_googleapi\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (D:\project1_googleapi\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\project1_googleapi\node_modules\express\lib\router\index.js:175:3)
    at router (D:\project1_googleapi\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\project1_googleapi\node_modules\express\lib\router\index.js:175:3)
    at router (D:\project1_googleapi\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at logger (D:\project1_googleapi\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at D:\project1_googleapi\node_modules\body-parser\lib\read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:206:9)
    at invokeCallback (D:\project1_googleapi\node_modules\raw-body\index.js:231:16)
    at done (D:\project1_googleapi\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (D:\project1_googleapi\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:518:28)
    at IncomingMessage.emit (node:domain:488:12)
    at endReadableNT (node:internal/streams/readable:1696:12)
    at processTicksAndRejections (node:internal/process/task_queues:82:21) 
    url - /api/v1/user/profile 
    method - GET 
    IP - ::1
   {"timestamp":"2024-01-31T07:16:50.483Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at D:\project1_googleapi\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (D:\project1_googleapi\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.verify (D:\project1_googleapi\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (D:\project1_googleapi\src\middlewares\/auth.middleware.js:22:32)
    at tryCatch (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at apply (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12)
    at userAuth (D:\project1_googleapi\src\middlewares\/auth.middleware.js:12:22)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\project1_googleapi\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (D:\project1_googleapi\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\project1_googleapi\node_modules\express\lib\router\index.js:175:3)
    at router (D:\project1_googleapi\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\project1_googleapi\node_modules\express\lib\router\index.js:175:3)
    at router (D:\project1_googleapi\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at logger (D:\project1_googleapi\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at D:\project1_googleapi\node_modules\body-parser\lib\read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:206:9)
    at invokeCallback (D:\project1_googleapi\node_modules\raw-body\index.js:231:16)
    at done (D:\project1_googleapi\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (D:\project1_googleapi\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:518:28)
    at IncomingMessage.emit (node:domain:488:12)
    at endReadableNT (node:internal/streams/readable:1696:12)
    at processTicksAndRejections (node:internal/process/task_queues:82:21) 
    url - /api/v1/user/profile 
    method - GET 
    IP - ::1
   {"timestamp":"2024-01-31T07:16:58.508Z"}
error: 
    status - 500 
    message - JsonWebTokenError: jwt must be provided
    at Object.verify (D:\project1_googleapi\node_modules\jsonwebtoken\verify.js:53:17)
    at _callee$ (D:\project1_googleapi\src\middlewares\/auth.middleware.js:22:32)
    at tryCatch (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at apply (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12)
    at userAuth (D:\project1_googleapi\src\middlewares\/auth.middleware.js:12:22)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\project1_googleapi\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (D:\project1_googleapi\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\project1_googleapi\node_modules\express\lib\router\index.js:175:3)
    at router (D:\project1_googleapi\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\project1_googleapi\node_modules\express\lib\router\index.js:175:3)
    at router (D:\project1_googleapi\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at logger (D:\project1_googleapi\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at D:\project1_googleapi\node_modules\body-parser\lib\read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:206:9)
    at invokeCallback (D:\project1_googleapi\node_modules\raw-body\index.js:231:16)
    at done (D:\project1_googleapi\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (D:\project1_googleapi\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:518:28)
    at IncomingMessage.emit (node:domain:488:12)
    at endReadableNT (node:internal/streams/readable:1696:12)
    at processTicksAndRejections (node:internal/process/task_queues:82:21) 
    url - /api/v1/user/profile 
    method - GET 
    IP - ::1
   {"timestamp":"2024-01-31T07:17:09.350Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at D:\project1_googleapi\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (D:\project1_googleapi\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.verify (D:\project1_googleapi\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (D:\project1_googleapi\src\middlewares\/auth.middleware.js:22:32)
    at tryCatch (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at apply (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12)
    at userAuth (D:\project1_googleapi\src\middlewares\/auth.middleware.js:12:22)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\project1_googleapi\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (D:\project1_googleapi\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\project1_googleapi\node_modules\express\lib\router\index.js:175:3)
    at router (D:\project1_googleapi\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\project1_googleapi\node_modules\express\lib\router\index.js:175:3)
    at router (D:\project1_googleapi\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at logger (D:\project1_googleapi\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at D:\project1_googleapi\node_modules\body-parser\lib\read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:206:9)
    at invokeCallback (D:\project1_googleapi\node_modules\raw-body\index.js:231:16)
    at done (D:\project1_googleapi\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (D:\project1_googleapi\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:518:28)
    at IncomingMessage.emit (node:domain:488:12)
    at endReadableNT (node:internal/streams/readable:1696:12)
    at processTicksAndRejections (node:internal/process/task_queues:82:21) 
    url - /api/v1/user/profile 
    method - GET 
    IP - ::1
   {"timestamp":"2024-01-31T07:17:22.906Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at D:\project1_googleapi\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (D:\project1_googleapi\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.verify (D:\project1_googleapi\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (D:\project1_googleapi\src\middlewares\/auth.middleware.js:22:32)
    at tryCatch (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at apply (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12)
    at userAuth (D:\project1_googleapi\src\middlewares\/auth.middleware.js:12:22)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\project1_googleapi\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (D:\project1_googleapi\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\project1_googleapi\node_modules\express\lib\router\index.js:175:3)
    at router (D:\project1_googleapi\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\project1_googleapi\node_modules\express\lib\router\index.js:175:3)
    at router (D:\project1_googleapi\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at logger (D:\project1_googleapi\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at D:\project1_googleapi\node_modules\body-parser\lib\read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:206:9)
    at invokeCallback (D:\project1_googleapi\node_modules\raw-body\index.js:231:16)
    at done (D:\project1_googleapi\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (D:\project1_googleapi\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:518:28)
    at IncomingMessage.emit (node:domain:488:12)
    at endReadableNT (node:internal/streams/readable:1696:12)
    at processTicksAndRejections (node:internal/process/task_queues:82:21) 
    url - /api/v1/user/profile 
    method - GET 
    IP - ::1
   {"timestamp":"2024-01-31T07:17:33.344Z"}
